gulpfile.js

```diff
@@ -0,0 +1,103 @@
+var gulp = require('gulp');
+var path = require('path');
+var concat = require('gulp-concat');
+var minifyCss = require('gulp-minify-css');
+var gutil = require('gulp-util');
+var argv = require('yargs')
+    .usage(
+      'Usage: $0 --appengine [Path to app engine]' +
+      '--clear_datastore [clear all data storage]' +
+      '--enable_sendmail [send emails]' +
+      '--minify [minify only in production]').argv;
+var gulpStartGae = require('./scripts/gulp-start-gae-devserver');
+var manifest = require('./manifest.json');
+
+var serveCmd = argv.appengine;
+var params = {
+  admin_host: '0.0.0.0',
+  admin_port: 8000,
+  host: '0.0.0.0',
+  port: 8181
+};
+
+var isMinified = (argv.minify == 'True');
+var dependencies = manifest.dependencies.thirdPartyStatic;
+var cssFilesPath = [];
+var fontFolderPath = [];
+var cssBackgroundPath = [];
+for (var dependencyId in dependencies) {
+  var dependency = dependencies[dependencyId];
+  if (dependency.hasOwnProperty('cssFiles')) {
+    var cssFiles = dependency.cssFiles;
+    for (var files in cssFiles) {
+      cssFilesPath.push(path.join('third_party', 'static',
+      dependency.targetDirPrefix + dependency.version,
+      cssFiles[files]));
+    }
+  }
+  if (dependency.hasOwnProperty('fontsPath')) {
+    var fontPrefix = '*.{eot,woff2,ttf,woff,eof,svg}';
+    fontFolderPath.push(path.join('third_party', 'static',
+      dependency.targetDirPrefix + dependency.version,
+      dependency.fontsPath, fontPrefix));
+  }
+  if (dependency.hasOwnProperty('cssBackgroundPath_image')) {
```

Comment:
Hm. Do we use these images, or do we currently do any sort of special-casing for them? If not, I would prefer not to introduce a new pattern just for that...